FlowCrypt/src/main/java/com/flowcrypt/email/service/CheckClipboardToFindKeyService.kt

```diff
@@ -145,7 +148,7 @@ class CheckClipboardToFindKeyService : Service(), ClipboardManager.OnPrimaryClip
         MESSAGE_WHAT -> {
           val clipboardText = msg.obj as String
           try {
-            val nodeKeyDetails = NodeCallsExecutor.parseKeys(clipboardText)
+            val nodeKeyDetails = PgpKey.parseKeys(clipboardText.toByteArray()).second
```

Comment:
These `.first` and `.second` are not obvious enough - we should find a way to refer to them in a more obvious way

Reply:
I've tried to not introduce extra classes and just use standard Pair class. This can be improved by adding a new data class with more readable field names.

Reply:
Alternatively we can completely drop returning format indication (I just tried to mimic what Typescrpt does) and return just key list, as `NodeCallsExecutor.parseKeys()` did.

Reply:
I don't see actually that such indication is used anywher - so maybe just drop it is best way to improve.

Reply:
> 
> 
> Looks good, adding a code clarity request.
> 
> Not sure about tests, our Android tests can be quite brittle when run in CI, so it's not clear if it passes or not sometimes until Den runs them locally. Hopefully over time it can be improved.

Yes, Android tests constantly fail here or there. Can we do something to improve that? Last time I've increased time limit and they've passed - maybe another increase needed?

Reply:
Data class is much, much better here. `PgpKey.parseKeys(clipboardText.toByteArray()).isArmored` and  `PgpKey.parseKeys(clipboardText.toByteArray()).keys` is much better than `.first` and `.second` from code clarity perspective. The added data class is always worth it.

Reply:
@tomholub @IvanPizhenko JUnit tests were broken. That's why tests were not passed.
I will be able to look at them in the morning.